PC/pyconfig.h

```diff
@@ -72,6 +72,9 @@ WIN32 is still required for the locale module.
 #define USE_SOCKET
 #endif
 
+#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY != WINAPI_FAMILY_DESKTOP_APP)
+#define MS_WINDOWS_UWP
```

Comment:
I am unsure how to name this macro since !desktop is not really the same as UWP.

Reply:
@zooba e.g. a game on the xbox would use `WINAPI_FAMILY_GAMES`. It does not provide these functions, but it is now UWP app. This was the reason I initially thought about going with `MS_WIN_DESKTOP_APP`.

Reply:
Yeah, I think to keep things fairly simple we just want a name that implies "Win32 but limited API access". It may mean some specific cases become blurry, and we may use workarounds on platforms that don't strictly need them, but it's better than a proliferation of preprocessor variables (especially since these ones will clutter up extension author namespaces).

Perhaps `MS_WINDOWS_NON_DESKTOP`? I think the only distinction right now is between old-style PCs and all the newer platforms (whiteboard, hololens, Xbox).